app/services/jwt_service.py

```diff
@@ -5,6 +5,8 @@
 
 class JwtService:
     async def verify_token(self, token: str) -> None:
+        print(settings.JWT_SECRET_KEY)
```

Comment:
do we need this print?

Reply:
good catch, no that should not be there. I'll remove it

Reply:
apparently that method isn't actually used anywhere though. I'm not exactly sure what I was debugging with it lol..